src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -107,9 +108,11 @@ public PersistentDataStore() {
         String title = (String) entity.getProperty("title");
         Instant creationTime = Instant.parse((String) entity.getProperty("creation_time"));
         Boolean isPrivate = (Boolean) entity.getProperty("is_private");
+        List<String> membersAsList = (List<String>) entity.getProperty("members");
 
         Conversation conversation = new Conversation(uuid, ownerUuid, title, creationTime);
         if(isPrivate != null && isPrivate == true) conversation.makePrivate();
+        if(membersAsList != null) conversation.addMembers(new HashSet<>(membersAsList));
```

Comment:
nit: (here and the previous line) I'd prefer breaking the if condition's body to next line, preferably with braces. It's much more readable that way, and avoids incorrect edits in the future.